Articles/ShortArticles/PullRequestSizeConsiderations.md

```diff
@@ -0,0 +1,155 @@
+discussed with the team ahead of submitting the changes.
+
+The upshot is that keeping PRs as small as reasonably practical is a best practice. Investing
+time up front to decide how to split large changes into manageable PRs that can be quickly
+integrated into a large code base is as important as the design and coding work that goes
+into the bug fixes and feature enhancements the changes are proposed for to begin with.
+
+Some literature uses the notion of a *reviewable unit of work*<sup>[20]</sup>
+to separate the concept of code review from how it is practiced in any particular tool.
+A reviewable unit could be a
+PR, a commit, or a single file's patch/diff.<sup>[21]</sup>
+In fact, long before GitHub introduced pull requests in 2008 (which was
+in turn based on Git's `request-pull`<sup>[22]</sup> operation),
+seasoned software professionals engaged in the conceptually equivalent processes using a
+patchwork of approaches and tools<sup>[23]</sup>
+(or using something like CodeStriker<sup>[24]</sup>) designed specifically for
+code review) which often even included emailing around patchfiles.
+
+Finally, it is worth pointing out that many commercial companies, including Google,
+don't use formal branching mechanisms commonly available in revision control systems like Git
+to manage code review. Of course, they still engage in code review but do so by other means and
+instead of using branches keep everything merged on a single, monolithic line of development<sup>[25]</sup>
+(e.g. *trunk* or *mainline*). In portions of the code base in transition, both new and old
+code paths commonly exist simultaneously, controlled through the use of conditional flags,
+a practice that is highly conducive to incorporating large changes in small, incremental pieces.
+
+[1]: https://sback.it/publications/icse2018seip.pdf  "Modern Code Review: A Case Study at Google {}"
+[2]: https://www.microsoft.com/en-us/research/wp-content/uploads/2016/02/bosu2015useful.pdf "Characteristics of Useful Code Reviews: An Empirical Study at Microsoft {}"
+[3]: https://www.microsoft.com/en-us/research/wp-content/uploads/2015/05/PID3556473.pdf "Code Reviews Do Not Find Bugs: How the Current Code Review Best Practice Slows Us Down {}"
+[4]: https://www.thedroidsonroids.com/blog/splitting-pull-request "How to Split Pull Requests – Good Practices, Methods and Git Strategies {}"
+[5]: https://derwolfe.net/2016/01/23/splitting-up-pull-requests/ "Splitting Up Pull Requests {}"
+[6]: https://glennstovall.com/5-ways-to-carve-large-pull-requests-into-bite-sized-ones/ "5 Ways to Carve Large Pull Requests Into Bite-Sized Ones {}"
+[7]: https://www.michaelagreiler.com/stacked-pull-requests/ "Stacked pull requests: make code reviews faster, easier, and more effective {}"
+[9]: https://jg.gg/2018/09/29/stacked-diffs-versus-pull-requests/ "Stacked Pull Requests {}"
+[10]: https://github.com/marketplace/stacked-pull-requests "Stacked Pull Requests {}"
```

Comment:
So here is the source of the misnamed (duplicate) "Stacked Pull Requests" reference in the file `PullRequestSizeConsiderations-wikized.md` file mentioned above that should be called "Stacked Diffs Versus Pull Requests"?